File: src/IsItHalloween.com/isithalloween.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string month, date;
    string str1 ("OCT 31");
    string str2 ("DEC 25");

    cin >> month >> date;
    string md = month + " " + date;
    if (md.compare(str1) == 0 || md.compare(str2) == 0)
        cout << "yup";
    else
        cout << "nope";
    return 0;
}

File: src/IsItHalloween.com/IsItHalloween.java
import java.io.*;

public class IsItHalloween {
    public static void main(String[] args) throws IOException {
        InputStreamReader inp = new InputStreamReader(System.in);
        BufferedReader sc = new BufferedReader(inp);
        PrintWriter writer = new PrintWriter(System.out);
        
        String date = sc.readLine();
        writer.print(date.equals("OCT 31") || date.equals("DEC 25") ? "yup" : "nope");

        writer.flush();
    }
}